src/controllers/courseController.ts

```diff
@@ -59,6 +59,9 @@ export class CourseController {
 		this.router.get('/content-management/courses/details/:courseId?', this.getCourseDetails())
 		this.router.post('/content-management/courses/details/:courseId?', this.setCourseDetails())
 
+		this.router.get('/content-management/courses/:courseId/sort-modules?', this.sortModules())
```

Comment:
this will math either `.../sort-module` or `.../sort-modules` -  was that the idea?